src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/FieldMaker.java

```diff
@@ -46,6 +46,7 @@ public static Field makeField(SQLExpr expr, String alias, String tableAlias) thr
             return makeField(makeBinaryMethodField((SQLBinaryOpExpr) expr, alias, true), alias, tableAlias);
 
         } else if (expr instanceof SQLAllColumnExpr) {
+            return new Field("*", "");
```

Comment:
Constant for "*"? Maybe even special field type?

Reply:
Sure, will add constant for now since I don't see different behavior worth creating a new domain class. Thanks!